src/Microsoft.AspNetCore.WebHooks.Receivers/WebHookAttribute.cs

```diff
@@ -24,28 +24,12 @@ public abstract class WebHookAttribute : Attribute, IAllowAnonymous, IFilterFact
     {
         private string _id;
 
-        // TODO: Move attribute constructors' comments, especially the recommended action signatures, to class level.
-        // TODO:  Do the same for all subclasses. As-is important information is hard to find.
-        /// <summary>
-        /// <para>
-        /// Instantiates a new <see cref="WebHookAttribute"/> indicating the associated action is a WebHook
-        /// endpoint for all enabled receivers.
-        /// </para>
-        /// <para>The signature of the action should be:
-        /// <code>
-        /// Task{IActionResult} ActionName(string receiverName, string id, string[] events, TData data)
-        /// </code>
-        /// or the subset of parameters required. <c>TData</c> must be compatible with expected requests.
-        /// </para>
-        /// <para>This constructor should usually be used at most once in a WebHook application.</para>
-        /// <para>
-        /// The default route <see cref="Mvc.Routing.IRouteTemplateProvider.Name"/> is <see langword="null"/>.
-        /// </para>
-        /// </summary>
-        protected WebHookAttribute()
+        internal WebHookAttribute()
         {
         }
 
+        // TODO: Move attribute constructors' comments, especially the recommended action signatures, to class level.
+        // TODO:  Do the same for all subclasses. As-is important information is hard to find.
```

Comment:
Can we use issues for these things instead of comments?

Reply:
How about a blanket "Address Core comments and TODOs" issue? There are a fair number of dangling `// TODO` and `// ???` comments. I'd rather do a (separate) single PR to do the work than create the issues and do an interim PR to remove the comments.

I'm just moving this particular TODO from one place to another.

Reply:
See #178